installation/linux.md

```diff
@@ -166,10 +181,7 @@ sudo systemctl start openhab2.service
 To uninstall openHAB 2 and get rid of all related files managed by the apt package manager, make a backup, then unistall openHAB and remove the repository:
 
 ```shell
-sudo apt-get purge openhab2-offline
-# respectively
-sudo apt-get purge openhab2-online
-
+sudo apt-get purge openhab2
```

Comment:
It would probably be wise to mention all packages here - at least in the transition phase. Not installed ones will be ignored but a user looking to "restart" may want to uninstall openhab2-offline first!?